UPGRADE-1.4.md

```diff
@@ -0,0 +1,178 @@
+### UPGRADE FROM 1.X TO 2.0
```

Comment:
The file name and this header does not correspond well together. Let's rename it to 2.0

Reply:
In 2.0 we drop deprecations only so the file name should stay but the header needs to be changed. Am I right?